src/test/java/test/postinstall/ElasticSearchInstallTest.java

```diff
@@ -0,0 +1,64 @@
+package test.postinstall;
+
+import io.fabric8.kubernetes.api.model.ContainerStatus;
+import io.fabric8.kubernetes.api.model.Node;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.api.model.apps.StatefulSet;
+import io.fabric8.kubernetes.api.model.metrics.v1beta1.NodeMetrics;
+import io.vavr.collection.Array;
+import io.vavr.collection.Map;
+import io.vavr.control.Option;
+import org.assertj.core.description.Description;
+import org.assertj.core.description.LazyTextDescription;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIf;
+import test.model.Product;
+
+import java.util.function.Consumer;
+
+import static io.restassured.RestAssured.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assumptions.assumeThat;
+import static org.hamcrest.Matchers.*;
+import static test.postinstall.Utils.*;
+
+@EnabledIf("isBitbucket")
+class ElasticSearchInstallTest {
+    static boolean isBitbucket() {
+        return productIs(Product.bitbucket);
+    }
+
+    private static final KubeClient client = new KubeClient();
+
+    // See helm_install.sh for where this host is generated.
+    static final String esIngressBase = "https://" + getRelease() + "-elasticsearch-master-0.kube.deplops.com";
```

Comment:
`kube.deplops.com` is that too hardcoded? Other K8s clusters will have different domains

Reply:
Good point. I'll make it configurable.

Reply:
Can you use the value from the `helm_parameters` file?

Reply:
Yep, that's what I've done.